src/EventLoop/Internal/DriverSuspension.php

```diff
@@ -87,7 +90,22 @@ public function suspend(): mixed
             $this->pending = false;
             $result && $result(); // Unwrap any uncaught exceptions from the event loop
 
-            throw new \Error('Event loop terminated without resuming the current suspension');
+            $info = '';
+            $suspensions = $this->suspensions->get();
+            if ($suspensions) {
+                \gc_collect_cycles();
+
+                foreach ($suspensions as $suspension) {
+                    if ($suspension->fiber === null) {
+                        continue;
+                    }
+
+                    $reflectionFiber = new \ReflectionFiber($suspension->fiber);
+                    $info .= "\n\n" . $this->formatStacktrace($reflectionFiber->getTrace(\DEBUG_BACKTRACE_IGNORE_ARGS));
```

Comment:
possibly check `ini_get("zend.exception_ignore_args")`? I like args :-D

Reply:
Yes, I actually dumped the args locally, so I'll improve this.